### PR TITLE
list-env CLI command

### PR DESCRIPTION
**Pull request by DongLieu**
Closes: #4527

## What is the purpose of the change

Add list-env for CLI

### CODE REVIEW

**Comment by DongLieu** [comment]
> Sweet, can you attach how the command looks like when actually ran on command line?

image

File: cmd/osmosisd/cmd/change_environment.go [hunk 1]
```diff
@@ -226,3 +226,54 @@ func customArgs(cmd *cobra.Command, args []string) error {
...
+
+			// OSMOSISD_ENVIRONMENT
+			val := os.Getenv(EnvVariable)
+			path, err = environmentNameToPath(val)
+			if err != nil {
+				fmt.Printf("Error: %v \n", err)
+			}
+
+			fmt.Println("Environment name: ", EnvVariable)
+			fmt.Println("Environment path: ", path)
```
Comment (mattverse) [inline cmd/osmosisd/cmd/change_environment.go:273]:
```suggestion
		         fmt.Printf("Environment name, path: %s, %s\n", EnvMainnet, path)

			// localnet
			path, err = environmentNameToPath(EnvLocalnet)
			if err != nil {
				fmt.Printf("Error: %v \n", err)
			}

			fmt.Printf("Environment name, path: %s, %s\n", EnvLocalnet, path)

			// testnet
			path, err = environmentNameToPath(EnvTestnet)
			if err != nil {
				fmt.Printf("Error: %v \n", err)
			}

			fmt.Printf("Environment name, path: %s, %s\n", EnvTestnet, path)
			
			// OSMOSISD_ENVIRONMENT
			val := os.Getenv(EnvVariable)
			path, err = environmentNameToPath(val)
			if err != nil {
				fmt.Printf("Error: %v \n", err)
			}

			fmt.Printf("Environment name, path: %s, %s\n", EnvVariable, path)
```

**Review by mattverse — APPROVED** [review]
Nice! LGTM, excited about this

**Review by czarcas7ic — APPROVED** [review]
LGTM, will merge on CI passing, thanks!

**Comment by czarcas7ic** [comment]
@DongLieu can you please fmt the file, I would do it but its not in a repo I have write access to

**Comment by DongLieu** [comment]
> @DongLieu can you please fmt the file, I would do it but its not in a repo I have write access to

fmt the file, you mean:
image

**Comment by DongLieu** [comment]
Or after commit change fmt:
image

**Pull request merged**